src/apps/relay/dbdrivers/dbd_pgsql.c

```diff
@@ -777,7 +777,10 @@ static void pgsql_reread_realms(secrets_list_t *realms_list) {
             if (rval) {
               get_realm(rval);
               ur_string_map_value_type value = strdup(rval);
-              ur_string_map_put(o_to_realm_new, (ur_string_map_key_type)oval, value);
+              int ret = ur_string_map_put(o_to_realm_new, (ur_string_map_key_type)oval, value);
+              if (ret == -1) {
+                free(rval);
```

Comment:
Thx for the fix @eakraly 
aren't we supposed to free `value` instead of `rval`?

Reply:
Right, why did I write this?